index.html

```diff
@@ -1543,227 +1560,439 @@
       <h3><a href="#h_numbers">Numbers</a>
       </h3>
 
+
       <section id="h_preferred_terminology">
-	<h4><a href="#h_preferred_terminology">Preferred Terminology</a></h4>
-	<p>Before entering this section in  we need to introduce few preferred terminological
-	  conventions for disambiguation and simplicity.</p>
-
-	<ul>
-	  <li><strong><span class="qterm">European Numerals</span></strong> are
-	    0, 1, 2, 3, 4, 5, 6, 7, 8, 9. They are also referred to as <span class="qterm">Western
-	      Arabic Numerals</span> or simply as <span class="qterm">Arabic Numerals</span>. Although
-	    these are terminologically correct terms, to avoid confusions we will refrain from using
-	    these phrases to refer to these numerals. <span class="qterm">European Numerals</span> or
-	    <span class="qterm">ASCII Digits</span> are used instead;
-	  </li>
-	  <li><strong><span class="qterm">Arabic-Indic Numerals</span></strong> are <bdi>٠</bdi>, <bdi>١</bdi>, <bdi>٢</bdi>,
-              <bdi>٣</bdi>, <bdi>٤</bdi>, <bdi>٥</bdi>, <bdi>٦</bdi>, <bdi>٧</bdi>, <bdi>٨</bdi>,
-              <bdi>٩</bdi>;</li>
-          <li><strong><span class="qterm">Eastern Arabic-Indic Numerals</span></strong> are ۰, ۱, ۲, ۳, ۴, ۵, ۶, ۷, ۸, ۹;</li>
-          <li><strong>Extended Arabic-Indic Numerals</strong> same as Eastern Arabic-Indic Numerals.</li>
-	  <li><strong>Western Arabic Numerals</strong> same as European Numerals;</li>
-	  <li><strong>Eastern Arabic Numerals</strong> is used to refer to both Arabic-Indic and
-	    Eastern Arabic-Indic Numerals. Should be avoided due to ambiguity;</li>
-	  <li><strong>Indic Numerals</strong> should be avoided to refer to either of Arabic-Indic
-	    or Eastern Arabic-Indic numerals.</li>
+        <h4><a href="#h_preferred_terminology">Preferred Terminology</a>
+        </h4>
+
+
+        <p>Before entering this section in we need to introduce few preferred terminological
+        conventions for disambiguation and simplicity.</p>
+
+
+        <ul>
+          <li><strong><span class="qterm">European Numerals</span></strong> are 0, 1, 2, 3, 4, 5,
+          6, 7, 8, 9. They are also referred to as <span class="qterm">Western Arabic
+          Numerals</span> or simply as <span class="qterm">Arabic Numerals</span>. Although these
+          are terminologically correct terms, to avoid confusions we will refrain from using these
+          phrases to refer to these numerals. <span class="qterm">European Numerals</span> or
+          <span class="qterm">ASCII Digits</span> are used instead;</li>
+
+
+          <li>
+            <strong><span class="qterm">Arabic-Indic Numerals</span></strong> are
+
+            <bdi>
+              ٠
```

Comment:
The spaces inside the `bdi` tags here need to be fixed.

https://www.w3.org/International/groups/arabic-layout/track/actions/104